vignettes/scenarios.Rmd

```diff
@@ -0,0 +1,228 @@
+---
+title: "Defining epidemic scenarios and comparison"
+output:
+  bookdown::html_vignette2:
+    fig_caption: yes
+    code_folding: show
+pkgdown:
+  as_is: true
+vignette: >
+  %\VignetteIndexEntry{Defining epidemic scenarios and comparison}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  chunk_output_type: console
+---
+
+This is a bare-bones vignette showing the basic functionality of _scenarios_. This package is a work in progress, and more functionality and documentation, including updates to this vignette, will be added in the near future.
+
+::: {.alert .alert-warning}
+**New to epidemic modelling in R?** It may help to read [The Epidemiologist R Handbook](https://epirhandbook.com/en/epidemic-modeling.html) for a primer on scenario modelling.
+:::
+
+::: {.alert .alert-primary}
+## Use case {-}
+
+To **compare** the outcomes of epidemic scenario models, such as the proportion of the population infected at a stage of the outbreak. Current use cases focus on comparing the cumulative proportion of individuals infected over the course of an epidemic wave, called the 'final size' of the epidemic.
+:::
+
+::: {.alert .alert-secondary}
+### What you need {-}
+
+  1. An epidemic model R function that produces data frames as outputs,
+  2. Parameters to the epidemic model, which are typically arguments to the model function.
+
+### What we assume {-}
+
+  1. That the epidemic scenarios are broadly comparable on a conceptual level.
+:::
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(scenarios)
+```
+
+## Components of an epidemic scenario model
+
+The _scenarios_ package is intended to help with the comparison of epidemic model scenarios.
+The _scenarios_ package defines the `scenario` class, which is an epidemic model definition with or without its associated data.
+Each `scenario` is expected to have:
+
+1. A model function, 
+2. The model parameters in the form of named function arguments, and
+3. The number of replicates to run.
+
+<!-- ::: {.alert .alert-secondary}
+### Example of epidemic scenario model
+
+::: -->
+
+## Defining scenarios for the final size of an epidemic
+
+<!-- Text about finalsize calculations here, with a box - link to finalsize website.
+Set the scene for when this sort of comparison is likely to be necessary --- new epidemic outbreak, quick comparison of potential outcomes wanted. -->
+
+This example considers the definition of scenario specifications for the outbreak of an epidemic in the U.K. population, and uses demography and social contact data from the POLYMOD survey [@mossong2008].
```

Comment:
.bib file needed for reference.

Reply:
Fixed in https://github.com/epiverse-trace/scenarios/pull/12/commits/a79d01394c6d621f4ae96e047028e4bd5c8ff1ad